conf/configuration.go

```diff
@@ -54,24 +54,20 @@ type Configuration struct {
 	SiteURL string           `json:"site_url"`
 	JWT     JWTConfiguration `json:"jwt"`
 	Mailer  struct {
-		MaxFrequency time.Duration `json:"max_frequency"`
-		Autoconfirm  bool          `json:"autoconfirm"`
-		Host         string        `json:"host"`
-		Port         int           `json:"port"`
-		User         string        `json:"user"`
-		Pass         string        `json:"pass"`
-		MemberFolder string        `json:"member_folder"`
-		AdminEmail   string        `json:"admin_email"`
-		Subjects     struct {
+		MaxFrequency   time.Duration `json:"max_frequency"`
+		Autoconfirm    bool          `json:"autoconfirm"`
+		Host           string        `json:"host"`
+		Port           int           `json:"port"`
+		User           string        `json:"user"`
+		Pass           string        `json:"pass"`
+		MemberFolder   string        `json:"member_folder"`
+		TemplateFolder string        `json:"template_folder"`
+		AdminEmail     string        `json:"admin_email"`
+		Subjects       struct {
 			Confirmation string `json:"confirmation"`
 			Recovery     string `json:"recovery"`
 			EmailChange  string `json:"email_change"`
 		} `json:"subjects"`
-		Templates struct {
```

Comment:
I don't think we should remove this. I think they should be able to specify defaults here. For instance, if they don't want to put the templates out on the site. 

```
Templates struct {
  ConfirmationDefault string
  ConfirumationPath string
  RecoveryDefault string
  RecoveryPath string
  EmailChangeDefault string
  EmailChangePath string
}
```

Or something like that. Maybe we'd have a `TemplatePath` param in that struct and then specify the names of the template.